examples/gcp-values.yaml

```diff
@@ -1,8 +1,13 @@
 pachd:
   image:
     tag: 1.12.3
+  exposeDockerSocket: false
```

Comment:
This should be the chart default?

Reply:
This is left over from testing different options. Cleaned up in e110085